dns-secondary-operator/src/charm.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Charm for dns-secondary."""
+
+import logging
+import subprocess  # nosec
+import time
+import typing
+
+import ops
+from charms.dns_transfer.v0 import dns_transfer
+
+import constants
+import events
+import exceptions
+import topology
+from bind import BindService
+
+logger = logging.getLogger(__name__)
+
+STATUS_REQUIRED_INTEGRATION = "Required integration with DNS primary not found"
```

Comment:
Do we really need every static string to have its own global variable ? I would wait that it is used in at least 2 places before doing that